x/gov/abci.go

```diff
@@ -17,6 +17,17 @@ func EndBlocker(ctx sdk.Context, keeper *keeper.Keeper) {
 
 	logger := keeper.Logger(ctx)
 
+	// delete the canceled proposals from store.
+	// move the min initial deposit of the proposal to community-pool and refund remaining deposits.
+	keeper.IterateCanceledProposalQueue(ctx, func(proposal v1.Proposal) (stop bool) {
+		keeper.DeleteProposal(ctx, proposal.Id)
```

Comment:
Can we do this op after https://github.com/cosmos/cosmos-sdk/pull/13010/files#diff-84432fb495de7d21f9244eb6cbcfe39fc23d2e9c2ed9020a21974216dc4c5cb8R27